zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -55,7 +53,7 @@ class Handlers(jsonGenerator: ZipkinJson, mustacheGenerator: ZipkinMustache, que
 
     def apply(response: Response) {
       response.setContentType(typ)
-      response.content = ChannelBuffers.wrappedBuffer(content)
+      response.content = Buf.ByteArray(content)
```

Comment:
Somewhere else in this file, this `content` is basically the result of an InputStream read into a byte array. Is there a shortcut to create a shareable Buf from an InputStream directly?